<launch>
    <node pkg="igvc_perception" type="ground_filter" name="ground_filter_node" output="screen"/>
    <node pkg="rosbag" type="play" name="play" output="screen" args="-q $(find igvc_perception)/bag/JESSII_RUN_SUN_3_2019-06-10-16-06-21.bag"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find igvc_perception)/rviz/pt_filter.rviz"/>
</launch>
